### PR TITLE
Bug 1219650 - Implement *test_call_log_groups.py* as an integration test in JavaScript

### PR DESCRIPTION
**Pull request by justinpotts**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=deea58cffbaab641c3d15514bed09a29d045ea00

File: apps/communications/dialer/test/marionette/lib/dialer.js [hunk 1]
```diff
@@ -69,6 +69,16 @@ Dialer.prototype = {
     this.client.apps.switchToApp(Dialer.URL, 'dialer');
   },
 
+  injectMockCallLogEntry: function(entry) {
+    window.wrappedJSObject.CallLogDBManager.add(entry);
```
Comment (JohanLorenzo) [inline apps/communications/dialer/test/marionette/lib/dialer.js]: This line will work only inside gaia. You'll have to put inside:
```js
client.executeScript(function() { window.wrappedJSObject.CallLogDBManager.add(entry); });
```

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d26bcfad51218e5c533f0b19c5fc3c20564c2b21

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d97f124da42d9f46429aa620a4b9ad1f1355449c

**Pull request closed without merging**